### ISSUE TITLE
noUnusedLocals incorrect error for imports used for types

### ISSUE THREAD

**Issue by speigg** [Microsoft/TypeScript#10207]
Using `noUnusedLocals:true` compiler option.

```ts
import {EventData} from 'observable'
export interface LoadEventData extends EventData {
    eventName: 'load',
    url: string
}
```

**Expected behavior:**
No errors

**Actual behavior:** 
'EventData' is declared but never used.

**Comment by sheetalkamat** [comment]
This seems to be fixed in typescript@next as well as typescript-2.0
